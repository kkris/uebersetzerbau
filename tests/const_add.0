f = fun x -> $42 + 1337 end;
