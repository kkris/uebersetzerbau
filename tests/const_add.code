f:
movq $2806, %rax
ret
