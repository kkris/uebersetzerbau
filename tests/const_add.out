ident f
=
fun
ident x
->
num 66
+
num 1337
end
;
